test/jpypetest/test_non_ascii_paths.py

```diff
@@ -0,0 +1,45 @@
+"""
+Regression test for https://github.com/jpype-project/jpype/issues/1194
+"""
+
+import unittest
+import jpype
+from pathlib import Path
+import shutil
+from multiprocessing import get_context, Queue, Process
+
+
+class TestNonAsciiPaths(unittest.TestCase):
```

Comment:
please derive from common.JPypeTestCase, then you would not need to take about a running JVM.

Reply:
Thanks, I should have noticed that. Would you be so kind to add this as a comment at the class declaration, please? Then nobody would later on think, it'd be a good idea to change that again.

Reply:
Yea no worries :D I've just added that comment.